app/src/main/java/org/greenstand/android/TreeTracker/managers/LocationUpdateManager.kt

```diff
@@ -120,3 +129,51 @@ fun Location?.accuracyStatus(): Accuracy {
         Accuracy.BAD
     }
 }
+
+class LocationDataCapturer(
```

Comment:
Can this be it's own file?

Reply:
For now,  I like to use the kotlin's flexibility to use one file to contain related classes. If the LocationDataCapturer becomes big to deserve its own file, we could create one.